components/pedigree/resources/src/main/resources/pedigree/view/templateSelector.js

```diff
@@ -92,8 +76,7 @@ define([
          * @private
          */
         _onTemplateSelected: function(event, pictureBox) {
-            //console.log("observe onTemplateSelected");
-            this.dialog.close();
+            $$('.pedigree-import-chooser .msdialog-close')[0].click();
```

Comment:
I don't have the dialogModal class object here to use its close() method If I use TemplateSelector class inside the PedigreeEditor class before I create dialogModal in it.